controllers/homeController.js

```diff
@@ -13,51 +13,68 @@ router.get('/', function(req, res) {
     res.render('home', {champions: championMap})
 });
 
-router.post('/', function(req, res) {
+router.post('/results', function(req, res) {
     var summonerName = encodeURI(req.body.summonerName)
     var summonerName2 = encodeURI(req.body.summonerName2)
 
-    Promise.all([getSummonerStats(summonerName), getSummonerStats(summonerName2)]).then( results => {
-        if (results.length  == 2) {
-            var summonerStats = results[0]
-            var summonerStats2 = results[1]
-            var commonChampions = commonChampionFinder.findCommonChampions(summonerStats, summonerStats2)
+    var statProms = Promise.all([getSummonerStats(summonerName), getSummonerStats(summonerName2)])
+    statProms.then(function(resArr) {
```

Comment:
im just curious, this should be able to be done in arrow notation still, do you know what results would look like if i go an error using arrow notation?